sorc/chgres_cube.fd/program_setup.f90

```diff
@@ -98,16 +94,71 @@ module program_setup
 !                                 These names will be used to identify
 !                                 the tracer records in the output files.
 !                                 Follows the convention in the field table.
+!                                 FOR GRIB2 FILES: Not used. Tracers instead taken
+!                                 from the varmap file. 
 ! tracers_input                   Name of each atmos tracer record in 
 !                                 the input file.  May be different from
 !                                 value in 'tracers'. 
+!                                 FOR GRIB2 FILES: Not used. Tracers instead taken
+!                                 from the varmap file. 
 ! use_thomp_mp_climo              When true, read and process Thompson
 !                                 MP climatological tracers.  False,
 !                                 when 'thomp_mp_climo_file' is NULL.
 ! vcoord_file_target_grid         Vertical coordinate definition file
 ! wltsmc_input/target             Wilting point soil moisture content
 !                                 input/target grids
-! 
+!
+! nsoill_out                      Number of soil levels desired in the output data. 
+!                                 chgres_cube can interpolate from 9 input to 4 output
+!                                 levels. 
+!                                 DEFAULT: 4    
+!
+! Variables that are relevant only for "grib2" input type:
+!
+! grib2_file_input_grid           REQUIRED. File name of grib2 input data.
+!                                 Assumes atmospheric and surface data are in a single 
+!                                 file. 
+!
+! varmap_file                     REQUIRED. Full path of the relevant varmap file. 
+!
+! external_model                  The model that the input data is derived from. Current
+!                                 supported options are: "GFS", "HRRR", "NAM", "RAP". 
+!                                 Default: "GFS"
+!
+! vgtyp_from_climo                If false, interpolate vegetation type from the input 
```

Comment:
We have high-res versions of the statsgo soil type data and igbp vegetation type available as part of the model grid generation process.  Do you require different datasets?  If so, we should consider making them available for all users.

Reply:
@GeorgeGayno-NOAA Yes, we use the 30 s BNU soil types from WRF and 15s MODIS vegetation types (also from WRF) in the HRRR.  Those are required for us to be able to directly compare FV3-LAM output to the operational HRRR to assess whether we're making progress with the RRFS toward implementation in 2023.